### PR TITLE
Update Input with Spinner #145

### PR DESCRIPTION
**Pull request by carlo**
See #145.

### CODE REVIEW

File: src/components/Form/InputRaw.js [hunk 1]
```diff
@@ -91,32 +97,49 @@ const InputRaw = (props) => {
...
+
+    return (
+      <Spinner
+        brand
+        className="slds-input__spinner slds-m-right_xx-small"
+        size="x-small"
+      />
+    );
+  };
+
```
Comment (PeterSiegl) [inline src/components/Form/InputRaw.js]: even shorter: `return showSpinner && <Spinner ... />`

File: src/components/Form/InputRaw.js [hunk 2]
```diff
@@ -91,32 +97,49 @@ const InputRaw = (props) => {
...
   const sldsClasses = [
-    { 'slds-input_bare': bare },
-    { 'slds-input': !bare },
+    { 'slds-has-input-focus': isFocused },
+    bare ? 'slds-input_bare' : 'slds-input',
```
Comment (PeterSiegl) [inline src/components/Form/InputRaw.js]: i suppose this should be `slds-input--bare`

Comment (carlo) [reply src/components/Form/InputRaw.js]: Nope, `.slds-input_bare` is correct, see https://lightningdesignsystem.com/components/input/

**Comment by carlo** [comment]
Thanks for the review, @PeterSiegl.

I'd like @fspoettel to take a look as well — I'd like to make sure I correctly understood his first task in #145 (the focus class).

File: src/components/Form/InputRaw.js (hunk 2 above)
Comment (PeterSiegl) [reply src/components/Form/InputRaw.js]: okay, ignore my comment: react-lds is already on 2.3.1

**Pull request merged**